src/scss/styles.scss

```diff
@@ -0,0 +1,135 @@
+@import "../../node_modules/bootstrap/scss/bootstrap";
+@import "variables";
+
+
+
+// Login page
+
+#login-container {
```

Comment:
Too much of custom classes are used for things you can do with bootstrap classes. A good example here is that all the properties stated in the login-container id can be achieved with the bootstrap class. 

This way it looks like you don't need bootstrap. The next step is to reduce the use of custom style and replace it with a bootstrap equivalent.